service-deployment/bootstrap/prometheus/cleanup.sh

```diff
@@ -19,10 +19,11 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-NAMESPACE=${NAMESPACE:-monitor}
-KUBECTL="kubectl  --namespace=\"${NAMESPACE}\""
+KUBECTL="kubectl "
 
-INSTANCES="daemonset/node-exporter"
+INSTANCES="daemonset/node-exporter
+deployment/prometheus-deployment
+configmap/prometheus-configmap"
 
 for instance in ${INSTANCES}; do
   eval "${KUBECTL} delete --ignore-not-found --now \"${instance}\""
```

Comment:
read ${instance} param

Reply:
kubectl instead of ${KUBECTL}. don't use eval, unless you have a good reason.

---
In reply to: [158426378](https://github.com/Microsoft/pai/pull/119#discussion_r158426378) [](ancestors = 158426378)